test-lit/Semantics/omp-nested01.f90

```diff
@@ -1,3 +1,5 @@
+!RUN: %S/test_errors.sh %s %flang %t
+!XFAIL: *
```

Comment:
Why?

Reply:
XFail? This test is commented out, so we ported it, but left it as a failing test.